File: android/src/main/java/com/lightningkite/kwift/time/DateAlone.shared.kt
package com.lightningkite.kwift.time

import java.util.*

fun DateAlone.set(other: DateAlone): DateAlone {
    this.year = other.year
    this.month = other.month
    this.day = other.day
    return this
}

fun DateAlone.format(clockPartSize: ClockPartSize): String = dateFrom(
    this,
    TimeAlone.noon
).format(clockPartSize, ClockPartSize.None)

File: ios/Kwift/com/lightningkite/kwift/time/DateAlone.shared.swift
//Package: com.lightningkite.kwift.time
//Converted using Kwift2

import Foundation



extension DateAlone {
    public func set(other: DateAlone) -> DateAlone {
        self.year = other.year
        self.month = other.month
        self.day = other.day
        return self
    }
    public func set(_ other: DateAlone) -> DateAlone {
        return set(other: other)
    }
}
 
 

extension DateAlone {
    public func format(clockPartSize: ClockPartSize) -> String {
        return dateFrom(self, TimeAlone.noon).format(clockPartSize, ClockPartSize.None)
    }
    public func format(_ clockPartSize: ClockPartSize) -> String {
        return format(clockPartSize: clockPartSize)
    }
}
 
